angular.json

```diff
@@ -128,5 +118,8 @@
       }
     }
   },
-  "defaultProject": "client"
+  "defaultProject": "client",
+  "cli": {
+    "analytics": false
+  }
```

Comment:
This was added manually? Or you tried to update angular project?

Reply:
no - when start first time the project from fork, Angular asking about permission for catching some analytica data from the project, I'm choose NO ... 
previously  angular.json has only    -  "defaultProject": "client",  
after my answer, it was added by Angular cli (v.11) 
  "cli": {
    "analytics": false
  }